src/main/java/org/openstreetmap/atlas/utilities/runtime/RunScript.java

```diff
@@ -20,20 +20,30 @@
 {
     private static final Logger logger = LoggerFactory.getLogger(RunScript.class);
 
-    public static void run(final String script)
+    public static void run(final String command)
     {
-        run(script, new ArrayList<>());
+        run(new String[] { command });
     }
 
-    public static void run(final String script, final List<RunScriptMonitor> monitors)
+    public static void run(final String command, final List<RunScriptMonitor> monitors)
+    {
+        run(new String[] { command }, monitors);
+    }
+
+    public static void run(final String[] commandArray)
```

Comment:
This was a much needed addition. Getting exec code right is tricky - this should help avoid future gotchas.